src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -176,11 +174,13 @@ else if (address != null) {
                     throw new RuntimeException("Unable to send requests.", ex);
                 }
                 if (gbfResponse != null && gbfResponse.isSuccess()) {
-                    logger.info("Ordered kits");
+                    for (Status kitStatus : gbfResponse.getStatuses()) {
+                        logger.info("Order status from order endpoint for " + kitStatus.getOrderNumber() + " is " + kitStatus.getOrderStatus());
```

Comment:
TBOS team was asking for some specific numbers about when various orders were placed, so explicitly logging the order number here will make it easier to answer questions.  Prior to this, one would have to grep the logs looking for above statements, and then re-grep to find a correlation in time between that log statement and an indication of success.  But one would also need to look at other log messages to ensure that the order wasn't filtered out due to a bad address.